packages/wrangler/templates/startDevWorker/ProxyWorker.ts

```diff
@@ -137,17 +139,46 @@ export class ProxyWorker implements DurableObject {
 					// errors here are network errors or from response post-processing
 					// to catch only network errors, use the 2nd param of the fetch.then()
 
-					void sendMessageToProxyController(this.env, {
-						type: "error",
-						error: {
-							name: error.name,
-							message: error.message,
-							stack: error.stack,
-							cause: error.cause,
-						},
-					});
-
-					deferredResponse.reject(error);
+					// we have crossed an async boundary, so proxyData may have changed
```

Comment:
Should the comment really be saying that the underlying Worker may have changed so we need to decide whether this error should be retried or not?

Reply:
I've updated the comment – does that convey what you mean?

Reply:
https://github.com/cloudflare/workers-sdk/pull/4867/commits/29b41f962fd8d3c618164a7d74aabe6ea67d49a6